### PR TITLE
Added Bucket Sort (#147)

### PR DESCRIPTION
**Pull request by ShubhamGupta577**
* **Adding Bucket Sort in C++**
* **Issue Resolved (#147 )**

### CODE REVIEW

**Comment by ShubhamGupta577** [comment]
@plazzy99 Please review this.

File: Code/C++/Bucket Sort.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#include <iostream>
```
Comment (plazzy99) [inline Code/C++/Bucket Sort.cpp:1]: add what is bubble sort at the top in multi-line comment

File: Code/C++/Bucket Sort.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    return 0;
+}
+
+
+/*OUTPUT
+Before Sorting
+0.1235 0.2101 0.4142 0.0476 0.2045 0.315
+
+After Sorting
+0.0476 0.1235 0.2045 0.2101 0.315 0.4142
```
Comment (plazzy99) [inline Code/C++/Bucket Sort.cpp:48]: time and space complexity must be added here.

**Review by plazzy99 — CHANGES_REQUESTED** [review]
also, update the readme for the issue raised

**Comment by ShubhamGupta577** [comment]
@plazzy99 Requested Changes are done.

**Review by plazzy99 — COMMENTED** [review]
Readme needs to be updated for making this pr ready to merge

**Comment by ShubhamGupta577** [comment]
@plazzy99 . I am not getting that for which readme you are asking me to update.

**Comment by tarun26091999** [comment]
> @plazzy99 . I am not getting that for which readme you are asking me to update.

searching_sorting in the algorithm folder

**Comment by ShubhamGupta577** [comment]
@plazzy99 Thanks for the clarification, and the requested change has been made.

**Comment by ShubhamGupta577** [comment]
@plazzy99 Please have a review for this.

**Comment by tarun26091999** [comment]
conflicts still persist.

**Comment by ShubhamGupta577** [comment]
@tarun26091999 Please review it now.
Sorry for the inconvenience caused due to this.

**Comment by tarun26091999** [comment]
its not resolved yet.

**Comment by ShubhamGupta577** [comment]
@tarun26091999 . As I am new to open source contribution, that's why I am getting that what is the exact issue in this. Can you please help me to resolve this issue?

**Comment by yasharth291** [comment]
merging this will cause breaking changes @tarun26091999

**Pull request closed without merging**